craft_store/auth.py

```diff
@@ -107,7 +108,10 @@ def __init__(
     @staticmethod
     def decode_credentials(encoded_credentials: str) -> str:
         """Decode base64 encoded credentials."""
```

Comment:
Maybe add something like:

```suggestion
        """Decode base64 encoded credentials.
        
        :raises errors.CredentialsNotParseable: when the credentials are incorrectly encoded.
        """
```